kernel/packages/shared/apis/SceneStateStorageController/BuilderServerAPIManager.ts

```diff
@@ -0,0 +1,247 @@
+    try {
+      this.setManifestOnServer(builderManifest, identity)
+    } catch (e) {
+      console.trace(e)
+    }
+    return builderManifest
+  }
+
+  async setManifestOnServer(builderManifest: BuilderManifest, identity: ExplorerIdentity) {
+    const queryParams = builderManifest.project.id + '/manifest'
+    const urlToFecth = `${this.getBaseUrl()}${queryParams}`
+
+    const body = JSON.stringify({ manifest: builderManifest })
+    const headers = this.authorize(identity, 'put', '/projects/' + queryParams)
+    headers['Content-Type'] = 'application/json'
+
+    let params: RequestInit = {
+      headers: headers,
+      method: 'PUT',
+      body: body
+    }
+
+    const response = await fetch(urlToFecth, params)
+    const data = await response.json()
+    return data
+  }
+
+  getBaseUrl(): string {
+    if (getDefaultTLD() === 'org') return BASE_PROJECT_URL
+    else return BASE_PROJECT_URL_ROPSTEN
+  }
+
+  addAssetsFromManifest(manifest: BuilderManifest) {
+    Object.entries(manifest.scene.assets).forEach((asset) => {
+      if (!this.assets.has(asset[0])) {
+        this.assets.set(asset[0], asset[1])
+      }
+    })
+  }
+
+  createEmptyDefaultBuilderScene(land: string, eth_address: string): BuilderManifest {
+    let today = new Date().toISOString().slice(0, 10)
+    let projectId = uuid()
+    let sceneId = uuid()
+    let builderProject: BuilderProject = {
+      id: projectId,
+      title: 'Builder ' + land,
+      description: 'Scene created from the explorer builder',
+      is_public: false,
+      scene_id: sceneId,
+      eth_address: eth_address,
+      rows: 1,
+      cols: 1,
+      created_at: today,
+      updated_at: today,
+      created_location: land
+    }
+
+    let builderScene: BuilderScene = {
+      id: sceneId,
+      entities: {},
+      components: {},
+      assets: {},
+      metrics: {
+        textures: 0,
+        triangles: 0,
+        materials: 0,
+        meshes: 0,
+        bodies: 0,
+        entities: 0
+      },
+      limits: {
+        textures: 10,
+        triangles: 10000,
+        materials: 20,
+        meshes: 200,
+        bodies: 300,
+        entities: 200
+      },
+      ground: {
+        assetId: 'da1fed3c954172146414a66adfa134f7a5e1cb49c902713481bf2fe94180c2cf',
+        componentId: 'b5edf28e-b4e4-4a27-b0ac-84b3d77eff8e'
+      }
+    }
+    let builderManifest: BuilderManifest = {
+      version: 10,
+      project: builderProject,
+      scene: builderScene
+    }
+    return builderManifest
+  }
+
+  private readonly AUTH_CHAIN_HEADER_PREFIX = 'x-identity-auth-chain-'
+
+  createHeaders(idToken: string) {
+    if (!idToken) return {}
+    const headers: Record<string, string> = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${idToken}`
+    }
+    return headers
+  }
+
+  authorize = (identity: ExplorerIdentity, method: string = 'get', path: string = '') => {
```

Comment:
```suggestion
  authorize(identity: ExplorerIdentity, method: string = 'get', path: string = '') {
```
Both ways are valid, but we should try to make it consistent.